content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -104,95 +109,67 @@ stringData:
+file, modify the `data` or `stringData` field in your original configuration
+file and apply the file to your cluster. You can edit an existing `Secret` object unless it is
+[immutable](/docs/concepts/configuration/secret/#secret-immutable)
 
-```
-secret/mysecret created
-```
+For example, if you want to change the password from the previous example to
+`birdsarentreal`, do the following:
 
-## Check the Secret
+1.  Encode the new password string:
 
-The `stringData` field is a write-only convenience field. It is never output when
-retrieving Secrets. For example, if you run the following command:
+    ```shell
+    echo -n 'birdsarentreal' | base64
+    ```
 
-```shell
-kubectl get secret mysecret -o yaml
-```
+1.  Update the `data` field with your new password string:
 
-The output is similar to:
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: mysecret
+    type: Opaque
+    data:
+      username: YWRtaW4=
+      password: YmlyZHNhcmVudHJlYWw=
+    ```
 
-```yaml
-apiVersion: v1
-data:
-  config.yaml: YXBpVXJsOiAiaHR0cHM6Ly9teS5hcGkuY29tL2FwaS92MSIKdXNlcm5hbWU6IHt7dXNlcm5hbWV9fQpwYXNzd29yZDoge3twYXNzd29yZH19
-kind: Secret
-metadata:
-  creationTimestamp: 2018-11-15T20:40:59Z
-  name: mysecret
-  namespace: default
-  resourceVersion: "7225"
-  uid: c280ad2e-e916-11e8-98f2-025000000001
-type: Opaque
-```
+1.  Apply the file to your cluster:
 
-The commands `kubectl get` and `kubectl describe` avoid showing the contents of a `Secret` by
-default. This is to protect the `Secret` from being exposed accidentally to an onlooker,
-or from being stored in a terminal log.
-To check the actual content of the encoded data, please refer to
-[decoding secret](/docs/tasks/configmap-secret/managing-secret-using-kubectl/#decoding-secret).
+    ```shell
+    kubectl apply -f <path-to-configuration-file>
+    ```
 
-If a field, such as `username`, is specified in both `data` and `stringData`,
-the value from `stringData` is used. For example, the following Secret definition:
+    The output is similar to:
 
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  name: mysecret
-type: Opaque
-data:
-  username: YWRtaW4=
-stringData:
-  username: administrator
-```
+    ```
+    secret/database-creds configured
+    ```
 
-Results in the following Secret:
+Kubernetes updates the existing `Secret` object, because you supplied a
+configuration file that had updated data with the same object name.
 
-```yaml
-apiVersion: v1
-data:
-  username: YWRtaW5pc3RyYXRvcg==
-kind: Secret
-metadata:
-  creationTimestamp: 2018-11-15T20:46:46Z
-  name: mysecret
-  namespace: default
-  resourceVersion: "7579"
-  uid: 91460ecb-e917-11e8-98f2-025000000001
-type: Opaque
-```
+## Delete a Secret {#clean-up}
 
-Where `YWRtaW5pc3RyYXRvcg==` decodes to `administrator`.
-
-## Clean Up
-
-To delete the Secret you have created:
+To delete a Secret, use `kubectl`:
 
 ```shell
-kubectl delete secret mysecret
+kubectl delete secret <secret-name>
```

Comment:
The original text looks good. As a task, we walk the readers through a process with a concrete example. In this task, we showed how to create a Secret named `mysecret` and now we show how to clean up.

This generalization is not needed.

Reply:
Okay, I can change it back to the original example